editor/editor_log.h

```diff
@@ -124,6 +133,9 @@ class EditorLog : public HBoxContainer {
 	// Warnings or Errors are encounetered.
 	Button *tool_button;
 
+	bool is_loading_state = false; // Used to disable saving requests while loading (some signals from buttons will try trigger a save, which happens during loading).
+	Timer *save_state_timer;
```

Comment:
Timer should be initialized to `nullptr`, but seeing as there are other non-initialized pointers, it's not very important for now.